### DISCUSSION TITLE
Welcome to castagne Discussions!

### DISCUSSION THREAD

**Discussion by panthavma** [panthavma/castagne#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RyanGosden** [comment]
This looks great! I'll definitely give it a try. Are there any plans to port this to godot 4 at some point?

**Reply by panthavma** [reply]
Hello ! Yes, the move to Godot 4 will happen at the same time as moving to a compiled language for performance (thanks to the constraints present since day 1). I can't say when it will happen yet, I'm still finishing up the last things of my PhD before being able to continue dev again.

**Reply by RyanGosden** [reply]
That's brilliant! A friend and I are looking at creating a simple fighter. We have had limited experience with Gotdot so this would be a great experience. I wish you all the luck with your PHD!
